scholarx/archive/2020/index.html

```diff
@@ -265,4 +275,49 @@ <h6>{{title}}</h6>
         {{/data}}
     </div>
 </script>
+
+<script type="text/html" id="templateMentees">
+    <table class="table table-striped">
+        <thead>
+          <tr>
+            <th scope="col">Image</th>
+            <th scope="col">Name</th>
+            <th scope="col">Mentor</th>
+            <th scope="col">University</th>
+          </tr>
+        </thead>
+        <tbody>
+        {{#data}}
+          <tr onclick="window.location='{{linkedin}}';">
+            <th scope="row">
+                <img src="images/mentees/{{image}}" class="avatar">
+            </th>
+                <td>{{name}}</td>
+                <td>{{mentor}}</td>
+                <td>{{university}}</td>
+          </tr>
+        {{/data}}
+        </tbody>
+      </table>
+</script>
```

Comment:
Let's move the static elements to the above html section and keep only the dynamic part (`table body` and `table rows`) in the template.

Reply:
Resolved with commit 152274a . Please review